src/julia-syntax.scm

```diff
@@ -3862,7 +3862,7 @@ f(x) = yt(x)
                                            v)))
                                    cvs)))
                `(new_opaque_closure
-                 ,(cadr e) (call (core apply_type) Union) (core Any)
+                 ,(cadr e) (top Bottom) (core Any)
```

Comment:
```suggestion
                 ,(cadr e) (call (core apply_type) (core Union)) (core Any)
```
(Bottom is not a name used in the system)

Reply:
I wonder whether we should just define it in Core though. It seems a bit weird to always have to call `apply_type` to get the bottom type in lowering. Seems like replacing it with a single `GlobalRef` would be preferable

Reply:
nah, this is pretty good